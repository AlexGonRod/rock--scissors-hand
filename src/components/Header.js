import React from "react";
import Score from './Score';

const Header = ({ score }) => {
  return (
    <div className="header">
      <div className="text">
        <span>Rock, Paper, Scissors</span>
      </div>
      <Score text='Score' className='score-box__score' score={score}/>
    </div>
  );
};

export default Header;
