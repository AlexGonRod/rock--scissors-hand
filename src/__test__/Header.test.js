import React from 'react';
import {shallow} from 'enzyme';
import '@testing-library/jest-dom'
import Header from '../components/Header';

describe('<Header />', () => {
    let wrapper = shallow(<Header />);
    beforeEach(() => {
        wrapper = shallow(<Header />);
    });
    test('<Header /> renders', () => {
        expect(wrapper).toMatchSnapshot();
    })
});