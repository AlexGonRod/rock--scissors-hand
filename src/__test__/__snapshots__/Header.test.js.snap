// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> <Header /> renders 1`] = `
<div
  className="header"
>
  <div
    className="text"
  >
    <span>
      Rock, Paper, Scissors
    </span>
  </div>
  <Score
    className="score-box__score"
    text="Score"
  />
</div>
`;
